File: DSA_using_Python/rotated_array.py
from typing import List


def rotate(nums: List[int], k: int):
        for i in range(k):
            nums = nums[1:]+nums[:1]
        return nums

nums = [1,2,3,4,5,6,7] 
k=3

nums = rotate(nums,k)
print(nums)

File: DSA_using_Golang/rotated_array.go
package main

func rotated(nums []int, k int) []int {

	for i := range k {
		i = 0
		var temp = nums[i]
		for j := range len(nums) - 1 {
			nums[j] = nums[j+1]
		}
		nums[len(nums)-1] = temp
	}
	return nums
}
